packages/block-library/src/heading/style.scss

```diff
@@ -0,0 +1,13 @@
+h1,
```

Comment:
The same question as for Paragraph block:

Are you sure it’s safe to set those values for all headings including front-end as part of the block definition?